### DISCUSSION TITLE
How to download the application in android

### DISCUSSION THREAD

**Discussion by RachitKhandaka** [z-huang/music#1244] [Q&A]
Can some one tell me how I can download this app from github into phone  (android)

**Comment by kaneryu** [comment]
[https://github.com/z-huang/InnerTune/releases/tag/v0.5.3](https://github.com/z-huang/InnerTune/releases/tag/v0.5.3)
This is the latest release, pick the platform that fits your phone. Or you can install it the reccomended way, through F-Droid.
